bot.py

```diff
@@ -84,11 +84,30 @@ async def daily_reminder(ctx, time):
 @client.command(aliases=["start-pomodoro"])
 async def pomodoro(ctx, time):
     # Set a timer for 25 minutes
+    time = 1500
     # Notify the user after 25 minutes
+    message = await ctx.send("Your countdown has been started for 25 minutes")
+    while True:
+        time -= 1
+        if time == 0:
+            await message.edit(content=("Time remaining {:02d}:{:02d}".format(0, 0)))
```

Comment:
Time remaining message is not required, as it would just appear and disappear instantly.